common/djangoapps/track/backends/logger.py

```diff
@@ -33,7 +33,16 @@ def __init__(self, name, **kwargs):
         self.event_logger = logging.getLogger(name)
 
     def send(self, event):
-        event_str = json.dumps(event, cls=DateTimeJSONEncoder)
+        try:
+            event_str = json.dumps(event, cls=DateTimeJSONEncoder)
+        except UnicodeDecodeError:
+            # Will throw UnicodeDecodeError incase when there are
+            # latin1 encoded characters in string.
+            # Example {'string': '\xd3 \xe9 \xf1'}
+            for key in event.iterkeys():
+                if isinstance(event[key], str):
+                    event[key] = event[key].decode('latin1')
```

Comment:
Can we add some kind of logging when this happens, so that we can try to figure out why this is happening/where this is coming from?  Perhaps the key and the value would be a start.